server/src/database/index.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { Sequelize } from 'sequelize';
+import dbCreds from '../configs/database';
+
+export const sequelize = new Sequelize(dbCreds.database, dbCreds.username, dbCreds.password, {
+  host: dbCreds.host,
+  dialect: 'postgres',
+  dialectOptions: {
+    ssl: {
+      require: true
+    },
+  },
+  define: {
+    timestamps: true,
+    underscored: true,
+    freezeTableName: true
+  },
+  logging: console.log
+});
+
+export const connectToDatabase = async (): Promise<void | NodeJS.Process> => {
+  try {
+    await sequelize.authenticate();
+    console.log('database connected');
+  } catch (error) {
+    console.log('database connecting failed', error);
+    return process.exit(1);
```

Comment:
Rather than immediately exiting here, could we instead reject the promise/throw an Exception?

Reply:
I think we could. Could you elaborate on that?

Reply:
```suggestion
    throw new Error('database connection failed');
```

I'm thinking of something like that. I would consider an error for the database connection to only be fatal in terms of this one database connection operation, rather than the entire application (to allow more fine-grained unit testing, for example).